src/main/java/io/conduit/DestinationStream.java

```diff
@@ -50,15 +49,15 @@ public DestinationStream(SinkTask task,
 
     @Override
     public void onNext(Destination.Run.Request request) {
-        log.debug("Writing record...");
+        Logger.get().info("Writing record...");
```

Comment:
This will be _very_ verbose if a pipeline is writing lots of records, we should avoid logs that are emitted for every record flowing through the pipeline. If we supported leveled logging this could be logged with the "trace" level.

So for now I propose just removing this.

Reply:
Good point, definitely too verbose.